src/main/java/de/chojo/repbot/commands/Info.java

```diff
@@ -93,6 +93,8 @@ private MessageEmbed getResponse(SlashCommandInteractionEvent event) {
 
             List<GithubProfile> profiles = new ArrayList<>();
             for (var contributor : contributors) {
+                if(ContributorType.BOT.equals(contributor.type)) continue;
```

Comment:
```suggestion
                if (ContributorType.BOT == contributor.type) continue;
```
Since this is an enum a reference check might be cleaner